packages/slice-machine/components/Preview/components/IframeRenderer/index.tsx

```diff
@@ -73,25 +73,27 @@ const IframeRenderer: React.FunctionComponent<IframeRendererProps> = ({
   }, [client, size, sliceView]);
 
   return (
-    <Flex
-      sx={{
-        justifyContent: "center",
-        borderTop: "1px solid #F1F1F1",
-        margin: "0 auto",
-        overflow: "auto",
-        ...iframeSizes[size],
-      }}
-    >
-      <iframe
-        ref={ref}
-        src={canvasUrl}
-        style={{
-          border: "none",
-          height: "100%",
-          width: "100%",
+    <Box sx={{ minHeight: "calc(100vh - 67px)", bg: "grey01" }}>
```

Comment:
I thing we can use "flex: 1" instead of calc no ? 

As we don't need the header to be static no ?